electrical/PCB Assembly/README.md

```diff
@@ -0,0 +1,655 @@
+    check that you are using the correct motor controller, terminal block, etc.
+    If all your connections are correct, you may have to test your solder
+    contact between the components on the board itself.
+
+5.  Repeat the steps above for each of the drive motor terminal blocks, labeled
+    J17-22.
+
+### Corner Motor Blocks
+
+Do the following procedure for the terminal blocks labeled J23-26. These
+correspond to the corner motors for the rover. Terminal blocks correspond to the
+motor controller outputs in the following manner:
+
+| Terminal Block Label | RoboClaw Board Label | Motor Output Channel |
+| -------------------- | -------------------- | -------------------- |
+| J23                  | RC4                  | M1                   |
+| J24                  | RC4                  | M2                   |
+| J25                  | RC5                  | M1                   |
+| J26                  | RC5                  | M2                   |
+
+1.  Connect the wires to the motor in the following manner
+
+    | Signal    | Terminal Block Label | Motor Connector Wire Color |
+    | --------- | -------------------- | -------------------------- |
+    | Motor (+) | M+                   | Red                        |
+    | Motor (-) | M-                   | Black                      |
+
+2.  The main difference between the drive and corner motor systems is that for
+    the corner system, we need to use the encoders. We want to test the voltage
+    division circuit used on the control board; this divider which will expects
+    a 0-5V signal from the absolute Hall effect encoder. To simulate the
+    encoder, connect the +5V terminal on the motor terminal block straight into
+    the ENA signal in the same terminal block.
+
+3.  Connect to the motor controller in the Basic Motion GUI. Under the General
+    settings tab (under Encoders), change the type of encoder from Quadrature to
+    Absolute. You should see that the encoder values change to a number
+    somewhere around 1600. As long as it is a fairly constant value and is in
+    the range of 1400-2000 then everything is working. If the value varies
+    wildly or is not in the 1400-2000 range, recheck that the OP-amp is
+    installed in the correct direction. If this number still isn't correct then
+    make sure you correctly installed all the resistors/capacitors in the
+    assembly steps.
+
+4.  Under the PWM tab, move the slide bar and verify that the motor spins
+    accordingly.
+
+5.  Repeat this process for all the corner motor terminal blocks, labeled
+    J23-26.
+
+## Raspberry Pi Install
+
+Next up is to verify that power to the Raspberry Pi is working. For this, you'll
+need a working operating system installed on the SD card. Take a moment now to
+follow to the
+[Software Install steps](https://github.com/nasa-jpl/open-source-rover/blob/master/Software/Software%20Steps.pdf)
```

Comment:
need to update this reference. Should just refer to the setup section in: https://github.com/nasa-jpl/osr-rover-code